opencontext/server/routes/settings.py

```
@@ -152,16 +181,16 @@ async def update_model_settings(
                 vlm_success = GlobalVLMClient.get_instance().reinitialize()
-                logger.info("VLM客户端重新初始化成功")
+                logger.info("VLM client reinitialized successfully")
                 embedding_success = GlobalEmbeddingClient.get_instance().reinitialize()
-                logger.info("嵌入客户端重新初始化成功")
+                logger.info("Embedding client reinitialized successfully")
                 if not vlm_success or not embedding_success:
                     raise HTTPException(status_code=500, detail="internal error: reinitialize LLM clients failed")
                 
             except Exception as e:
-                logger.error(f"重新初始化LLM客户端失败: {e}")
+                logger.error(f"Failed to reinitialize LLM client: {e}")
```

Ruff (0.13.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)